src/main/kotlin/com/supergrecko/questionbot/commands/ManageCommands.kt

```diff
@@ -57,6 +57,30 @@ fun manageCommands(config: ConfigService, logService: LogService) = commands {
         }
     }
 
+    command("setLog") {
```

Comment:
Could you make the name lowercase for consistency?

Reply:
sure, makes sense